electron/main.js

```diff
@@ -169,13 +169,17 @@ function showMainWindow() {
     main.setBounds(init.restore('bounds', main.getBounds()));
   }
 
-  main.webContents.session.setCertificateVerifyProc((hostname = '', certificate = {}, cb) => {
-    const {data: cert = ''} = certificate;
+  main.webContents.session.setCertificateVerifyProc((request, cb) => {
+    const { hostname = '', certificate: { data: cert = '' } = {}, error } = request;
+
+    if (error !== undefined) {
+      return cb(0);
```

Comment:
If pinning checks out, we should probably do

`return cb(-3);`

I think it matches what the old `return cb(true)` did.